config/config-example.js

```diff
@@ -228,6 +228,11 @@ exports.repl = true;
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// disablehotpatchall - disables allowing hotpatching every hotpatchable part
+// of PS at once. Generally speaking, there probably won't be a need to for this,
+// which is why there's an option to disable this.
```

Comment:
>(I'm asking you to change this description)

honestly didn't realize that until i saw that comment and I was really confused, lol